python/audit/management/commands/clearaudit.py

```diff
@@ -140,11 +140,25 @@ def __prepare_csvs(self, *querysets, base_dir):
                 continue
 
             tmp_cvf_name = self.__get_csv_name(qs, now, base_dir)
-            with open(tmp_cvf_name, 'wt', newline='', encoding=self.encoding) as csv_file:
+            exists = os.path.exists(tmp_cvf_name)
```

Comment:
`pathlib` comes to replace `os.path` and `is_file` is better than `exists`

Reply:
done